packages/ai/mcp-stdio/mcp-stdio-transport.test.ts

```diff
@@ -117,12 +117,12 @@ describe('StdioMCPTransport', () => {
     });
 
     it('should handle child_process import errors', async () => {
```

Comment:
non-blocking: `should handle child_process import errors` is a legacy test (from before we moved stdio transport files out of the `ai/core` folder)! we can remove